src/mpid/ch4/netmod/ucx/ucx_init.c

```diff
@@ -332,11 +338,15 @@ int MPIDI_UCX_mpi_finalize_hook(void)
 }
 
 /* static functions for MPIDI_UCX_post_init */
+static void flush_cb(void *request, ucs_status_t status)
```

Comment:
Looks like this is accidentally added back in this commit.

Reply:
Good catch.